src/services/repository/DraftRepository.php

```diff
@@ -172,6 +173,9 @@ public function applyTranslationDraft($fileId, $file='', $draft='')
             throw new NotFoundHttpException('Draft not found');
         }
 
+        // this will add and enable any site missing in enabled sites of element only if section propagation
+        $this->enableForAllSupportedSites($file);
+
```

Comment:
@shnsumit would it be better to move the conditional `$element->getSection()->propagationMethod === Section::PROPAGATION_METHOD_CUSTOM` check here so that we are not triggering `enableForAllSupportedSites()` each time?

Reply:
@sidedwards  Actually to add that condition here we need to move many lines of code here to keep that out I have added if condition in function.

<img width="710" alt="image" src="https://user-images.githubusercontent.com/32937137/121418347-7c1de000-c988-11eb-93c0-4e84eb80e14d.png">

Let me know if that make sense other wise I will change it.

Reply:
@shnsumit okay, makes sense. Let's keep it the way it is.